hexa/pipelines/tests/test_schema.py

```diff
@@ -516,3 +521,134 @@ def test_delete_pipeline_version(self):
             {"success": True, "errors": []},
             r["data"]["deletePipelineVersion"],
         )
+
+    def test_add_pipeline_recipients(self):
+        self.test_create_pipeline_version()
+        pipeline = Pipeline.objects.filter_for_user(user=self.USER_ROOT).first()
+
+        r = self.run_query(
+            """
+            mutation updatePipeline($input: UpdatePipelineInput!) {
+                updatePipeline(input: $input) {
+                    success
+                    errors
+                    pipeline {
+                        recipients {
+                            user {
+                                id
+                            }
+                        }
+                    }
+                  }
+            }
+        """,
+            {
+                "input": {
+                    "id": str(pipeline.id),
+                    "recipientIds": [str(self.USER_ROOT.id)],
+                }
+            },
+        )
+        self.assertEqual(
+            {
+                "success": True,
+                "errors": [],
+                "pipeline": {"recipients": [{"user": {"id": str(self.USER_ROOT.id)}}]},
+            },
+            r["data"]["updatePipeline"],
+        )
+
+    def test_pipeline_new_run_with_recipients(self):
```

Comment:
I don't see the difference between this one and the `test_pipeline_new_run`.

Reply:
I would add one test : What happens when you try to add a non workspace's member as a recipient ?